### ISSUE TITLE
ask for clarification ch27

### ISSUE THREAD

**Issue by victorhck** [iggredible/Learn-Vim#95]
Hello.

Translating the [section Dictionary](https://github.com/iggredible/Learn-Vim/blob/master/ch27_vimscript_functions.md#dictionary) in chapter 27 I don't undertand what means when you say *namespace*. Don't know how to translate that word, because don't understand completly the second example...

Thanks for your support

**Comment by iggredible** [comment]
Good question! A namespace is used when you want to group different variables together. A namespace is used to avoid naming conflicts.

If you have a large collection of functions, you'll eventually run into name conflicts sooner or later.

For example, I share name with many people like Igor Stravinsky (Composer) and Igor (from Dr. Frankenstein). If inside a file I have 3 different Igor methods, there will be naming conflicts. To solve that, we can create a namespace to "organize" the Igor functions, something like:
- `composer.igor`
- `fiction.igor`
- `vimUser.igor`

In short, it's a way to organize variables / functions to make it cleaner and to avoid future conflicts.

And yes, `MyFar` is a typo. I'll fix that. Good catch!

**Comment by victorhck** [comment]
It's clear enough! Thanks a lot `vimUser.Igor` !! :þ

I think that should add a explanation like this in my translation as a side note.

**Comment by iggredible** [comment]
Glad I can help!
